packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal.tsx

```diff
@@ -30,14 +29,9 @@ const BuySellModal = () => {
     const [error_message, setErrorMessage] = React.useState('');
     const [is_submit_disabled, setIsSubmitDisabled] = React.useState(false);
     const [is_account_balance_low, setIsAccountBalanceLow] = React.useState(false);
-    const submitForm = React.useRef<(() => void) | null>(null);
 
     const show_low_balance_message = !is_buy_advert && is_account_balance_low;
 
-    const is_fixed_rate = buy_sell_store?.advert?.rate_type === ad_type.FIXED;
-
-    const setSubmitForm = (submitFormFn: () => void) => (submitForm.current = submitFormFn);
-
```

Comment:
deleted this, and instead added a new observable to pass the function to it, reason why is because when the exchange rate changes, it will unmount the Modal, resetting the submitForm.current value back to null, hence why they are getting issues with not being able to click on Confirm. 

Wouldve been better if we could bring the formik values up into the parent and then use useFormikContext, but that would require an entire overhaul of the BuySellModal, BuySellForm, BuySellFooter, which would take too long. This will all be better in v2